jax/source_info_util.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
```

Comment:
I assume this file is not meant to be part of the JAX public API, although people will want to use it, especially with the fast traceback implementation. Should we put it under a `_private` subdirectory? We should do something like that for all private modules eventually, but at least as we create new modules maybe we can start doing it.

Reply:
That makes sense, but
1. underscores in our names haven't stopped people from using them (see e.g. the recent uses of `pxla._thread_local_state`, and
2. an incremental approach where we only put some things we want to be private in `_private` might then imply things that aren't in `_private` are in fact public!

For the second reason, combined with not wanting to figure out a `_private` directory structure right now, I think we should punt on this.